src/gui/clips/ClipView.cpp

```diff
@@ -969,9 +968,9 @@ void ClipView::mouseMoveEvent( QMouseEvent * me )
 						TimePos position = (pClip->startTimeOffset() + positionOffset) % patternLength;
 						pClip->setStartTimeOffset(position);
 					}
-					else if (aClip)
+					else if (sClip || aClip || mClip)
```

Comment:
```suggestion
					else
```
This is always true when it isn't a pattern clip.